src/LanguageManager/LanguageManagerServiceProvider.php

```diff
@@ -0,0 +1,106 @@
+<?php # -*- coding: utf-8 -*-
+
+declare( strict_types = 1 );
+
+namespace Inpsyde\MultilingualPress\LanguageManager;
+
+use Inpsyde\MultilingualPress\Common\Admin\SettingsPage;
+use Inpsyde\MultilingualPress\Common\Nonce\WPNonce;
+use Inpsyde\MultilingualPress\Service\BootstrappableServiceProvider;
+use Inpsyde\MultilingualPress\Service\Container;
+use Inpsyde\MultilingualPress\Common\Type\Language;
+
+use function Inpsyde\MultilingualPress\get_available_languages;
+
+/**
+ * Service provider for all language manager objects.
+ *
+ * @package Inpsyde\MultilingualPress\LanguageManager
+ * @since   3.0.0
+ */
+final class LanguageManagerServiceProvider implements BootstrappableServiceProvider {
+
+	/**
+	 * Registers the provided services on the given container.
+	 *
+	 * @since 3.0.0
+	 *
+	 * @param Container $container Container object.
+	 *
+	 * @return void
+	 */
+	public function register( Container $container ) {
+
+		$container['multilingualpress.language_manager_settings_page'] = function ( Container $container ) {
+
+			return SettingsPage::with_parent(
+				SettingsPage::ADMIN_NETWORK,
+				SettingsPage::PARENT_NETWORK_SETTINGS,
+				__( 'Language Manager', 'multilingual-press' ),
+				__( 'Language Manager', 'multilingual-press' ),
+				'manage_network_options',
+				'language-manager',
+				$container['multilingualpress.language_manager_settings_page_view']
+			);
+		};
+
+		add_action( LanguageManagerSettingsPageView::CONTENT_DISPLAY, function() use ( $container ) {
+
+			$separator = new LanguageUsageList( $container['multilingualpress.languages'] );
+
+			$table = new LanguageListTable( $separator->get_by( LanguageUsageList::ACTIVE ) );
```

Comment:
This should be added as service to the container. You can still get the `$separator` off the container, and call methods on it.